aqua/qt55/Portfile

```diff
@@ -142,7 +142,7 @@ array set modules {
         {"Qt 3D"}
         "technology preview modules"
         "variant overrides: "
-        "revision 3"
+        "revision 4"
```

Comment:
I have not ruled out whether anyone may have successfully built the bundled assimp, so incrementing the revision anyway.